src/pages/log/LogListPage.tsx

```
@@ -0,0 +1,160 @@
+    useEffect(() => {
+        if (tripId === null) {
+            alert('잘못된 여행 id입니다.');
+            navigate(-1);
+        }
+    }, [tripId, navigate]);
+
+    if (tripId === null) return null;
+
+    const fetchLogs = useCallback(
+        async (reset = false) => {
+            if (isFetchingRef.current) return;
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)